Solutions/SonraiSecurity/Data Connectors/Connector_REST_API_Sonrai.json

```diff
@@ -0,0 +1,104 @@
+{
+  "id": "SonraiDataConnector",
+  "title": "Sonrai Data Connector",
+  "publisher": "Sonrai",
+  "descriptionMarkdown": "Use this data connector to integrate with Sonrai Security and get Sonrai tickets sent directly to Azure Sentinel.",
+  "graphQueries": [
+    {
+      "metricName": "Sonrai Tickets",
+      "legend": "Sonrai_Ticket_Logs",
+      "baseQuery": "Sonrai_Ticket_Logs"
```

Comment:
Can you please tell us what is Sonrai_Ticket_Logs? If it is a custom table please provide sample data.

Reply:
Changed Sonrai_Ticket_Logs to Sonrai_Tickets_CL